frontend/packages/integration-tests-cypress/views/labels.ts

```diff
@@ -0,0 +1,6 @@
+export const labels = {
+  inputLabel: (label: string) => cy.byTestID('tags-input').type(label),
+  confirmDetailsPageLabelExists: (label: string) => cy.get('.co-m-label__key').contains(label),
+  clickDetailsPageLabel: () => cy.get('.co-m-label__key').click(),
+  chipExists: () => cy.get('.pf-c-chip-group__label').should('exist'),
```

Comment:
Ideally we'd add a test ID here as well if possible.

Reply:
This one is a PF component and it doesn't look like I can pass one in.